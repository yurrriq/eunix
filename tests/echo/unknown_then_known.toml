bin.name = "echo"
args = ["-x", "-n", "unknown", "then", "known"]
binary = true
