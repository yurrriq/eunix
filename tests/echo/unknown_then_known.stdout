-x -n unknown then known
